clients/web/src/lib/components/atoms/form/Select/Select.svelte

```diff
@@ -0,0 +1,159 @@
+<script lang="ts" context="module">
+    export type SelectOptionValue = string | number;
+    export type SelectOption<T extends SelectOptionValue> = {label: string; value: T};
+    export type SelectOptions<T extends SelectOptionValue> = SelectOption<T>[];
+</script>
+
+<script lang="ts">
+    import {slide} from "svelte/transition";
+    import {nanoid} from "nanoid";
+
+    import type {FormControlSize, FormControlState} from "../form.types";
+    import {Icon} from "@steeze-ui/svelte-icon";
+    import {ChevronDown} from "@steeze-ui/heroicons";
+
+    // Define generic so that options and value have the same type
+    type T = $$Generic<SelectOptionValue>;
+    interface $$Props {
+        label: string;
+        value?: T;
+        options: SelectOptions<T>;
+
+        size?: FormControlSize;
+        placeholder?: string;
+        disabled?: boolean;
+        state?: FormControlState;
+        error?: string;
+    }
+
+    export let label: string;
+    export let value: T | undefined = undefined;
+    export let options: SelectOptions<T> = [];
+
+    export let size: FormControlSize = "md";
+    export let placeholder: string | undefined = undefined;
+    export let disabled: boolean = false;
+    export let state: FormControlState = "none";
+    export let error: string | undefined = undefined;
+
+    const labelId = `select_${nanoid()}`;
+    const errorId = `select-error_${nanoid()}`;
+</script>
+
+<div class="size-{size} state-{state}" class:disabled>
+    <label for={labelId}>{label}</label>
+
+    <div class="select-wrapper">
+        <select
+            id={labelId}
+            class:placeholder={placeholder && !value}
+            bind:value
+            aria-describedby={error ? errorId : undefined}
+            {disabled}
+        >
+            {#if placeholder}
+                <option value={undefined} disabled selected hidden>{placeholder}</option>
+            {/if}
+
+            {#each options as option}
+                <option value={option.value}>{option.label}</option>
+            {/each}
+        </select>
+
+        <span class="icon">
+            <Icon class="h-5/10" src={ChevronDown} />
+        </span>
+    </div>
+
+    {#if error && state === "invalid"}
+        <span class="error" id={errorId} transition:slide>{error}</span>
+    {/if}
+</div>
+
+<style lang="postcss">
+    /* General styling */
+    label {
+        @apply block mb-2 text-sm font-medium text-gray-300;
+    }
+
+    .select-wrapper {
+        @apply relative;
+    }
+
+    .icon {
+        @apply absolute h-full top-0 flex items-center pointer-events-none;
+    }
+
+    .disabled {
+        @apply text-gray-400;
+    }
```

Comment:
Should be moved to `class` attributes

Reply:
[998169d](https://github.com/SprocketBot/sprocket/pull/386/commits/998169dd32cdee569156d2748adf2862c2e99755)